src/lib/azure/azurecredentials.ts

```diff
@@ -0,0 +1,27 @@
+import { ClientSecretCredential } from "@azure/identity";
+import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
+import { azureClientId, azureClientSecret, azureTenantId } from "../../config";
```

Comment:
FYI, will be updating this to use the new config, shortly

Reply:
ok.